cocos/gfx/base/define.ts

```diff
@@ -2163,6 +2161,10 @@ export function IsPowerOf2 (x: number): boolean {
     return x > 0 && (x & (x - 1)) === 0;
 }
 
+function ceil (x: number): number {
+    return Math.ceil(x);
+}
```

Comment:
It introduces a function call. So i don't think it is a good idea to do it like this to reduce pacakge size.

Reply:
UPDATED:

Use function reference instead of defining a function for Math.ceil and Math.max
```ts
const ceil = Math.ceil;
```

Reply:
https://github.com/cocos/cocos-engine/pull/17529/commits/5035f0b59c007c7385865e589d0a13b59f6fc3fb